templates/admin/judge/contest/change_form.html

```diff
@@ -5,6 +5,10 @@
     <script>
         django.jQuery(function ($) {
             $('.reratelink').appendTo('div#bottombar').show();
+            $('.rejudgelink').click(function () {
+                if (confirm('{{ _('Are you sure you want to rejudge ALL the submissions?') }}'))
+                    location.href = this.getAttribute("href");
+            });
```

Comment:
Don't you want to prevent the default behaviour? Because if you don't prevent the default behaviour, when you press cancel the operation goes through anyways. In fact, just `return confirm('...')` for the entire handler would have sufficed.